### ISSUE TITLE
Missing inputs.url Component and other expected URL syntax

### ISSUE THREAD

**Issue by relayer464** [TiagoSilvaPereira/vemto-issues#189]
Greetings,

**Problem 1: Missing inputs.url on Create and Edit Forms**

The "Url" input type and other URL syntax does not appear to be working.  When I select this Input type, I essentially get what appears to be the same thing you would get by select the Input type of Text.

1. Create a CRUD that includes an input type "Url"
2. Generate project
3. Navigate to Create or Edit forms and get the following error message:

> InvalidArgumentException
> Unable to locate a class or view for component [inputs.url]. (View: C:\laragon\www\vemto_bookmarks\resources\views\app\bookmarks\form-inputs.blade.php)
> http://vemto_bookmarks.localhost/bookmarks/1/edit

4. When I navigate to **resources\views\components\inputs** there is no **url.blade.php** file.

Note: As a temporary work around,  I copied the text.blade.php file and modified the name some syntax.  After doing so, I don't get the error, but I assume this file _should_ have been generated.

**Problem 2: No href link** 

1. Navigate to the Index or Show page
2. Use or Url option doesn't generate a clickable link (just text)

**Expected behavior**

- I had expected that by using the Input Type of Url there would be a corresponding component and not an error
- I thought that by using the Input Type of URL I would get clickable links on the Index and Show pages

I don't think I've overlooked any kind of setting?  If so, I apologize.  Your advice and hard work are much appreciated.  :)

**Comment by TiagoSilvaPereira** [comment]
Hi @relayer464 thank you for reporting, I'll check and fix it ASAP

**Comment by 949mac** [comment]
Hi @TiagoSilvaPereira,

I think I just experienced the same issue.

![image](https://user-images.githubusercontent.com/23151962/124522105-e0c13480-dda6-11eb-95f8-06b93ae1282e.png)

![image](https://user-images.githubusercontent.com/23151962/124522108-e28af800-dda6-11eb-9eab-6327e46845fa.png)

**Comment by TiagoSilvaPereira** [comment]
@relayer464 @949mac fixed on 0.9.8

**Comment by relayer464** [comment]
Greetings,

Thanks, I see the URL input and index page link were fixed on the latest update.  I just regenerated a page that uses the Url input type and those two issues appear to be resolved.   

However, I noticed that the URL field on the "show" page is standard still just text.  It seems like it should also be a clickable link just like just like the update you made for the Index page.

Thanks!
